src/commands.py

```diff
@@ -257,11 +257,11 @@ def secondsToHuman(seconds):
 			if not "groups" in self.wrapper.permissions: self.wrapper.permissions["groups"] = {}
 			if not "users" in self.wrapper.permissions: self.wrapper.permissions["users"] = {}
 			if not "Default" in self.wrapper.permissions["groups"]: self.wrapper.permissions["groups"]["Default"] = {"permissions": {}}
-			if player.isOp():
+			if True:
```

Comment:
No, I don't think so.. I saw that too, but was lazy since it does nothing.  I think it is left over code from some debugging